org.eclipse.xtext.maven.plugin/pom.xml

```diff
@@ -110,6 +115,21 @@
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<!-- Needed by some of our IT projects:
```

Comment:
wonder why these are not accessible if just mention in the tests pom

Reply:
is this cause of upstream repos not availble in mono build

Reply:
My intention and understanding of integration tests for maven plugins are that our artifacts should NOT be downloaded from remote repositories; this way, we're sure we're testing the integration projects with our current artifacts (the ones built from the current sources). Now that we're on a single project, this makes sense.

That's why we use the invoker plugin to create a self-contained .m2 repo with the needed Xtext artifacts.
Note that in the IT projects I've disabled completely sonatype snapshots, I only left lsp4j taken from JIRO.
This way, we're sure that we are self-contained concerning our artifacts.

In general, despite split or mono repo, we made a terrible mistake in not using another .m2 repository for running integration tests, risking of corrupting the current .m2 repo.

Does that answer your question? Or did you mean something else?